modules/rubiconBidAdapter.js

```diff
@@ -311,11 +311,6 @@ export const spec = {
 
       applyFPD(bidRequest, VIDEO, data);
 
-      // if storedAuctionResponse has been set, pass SRID
```

Comment:
Actually, I do have a request here. In order to remove this code, I need the adapter to read ortb2Imp and merge it into the PBS request.

The adunit will contain, for example:
```
      ortb2Imp: {
        ext: {
          prebid: {
            storedauctionresponse: { id: "sample_video_response" }
          }
        }
      },
``` 
The video 'fork' of the rubicon adapter needs to pass this on imp[].ext.prebid.storedauctionresponse.id. It doesn't currently. 

Reply:
I'd feel more comfortable leaving this change to a magnite developer; they can feel free to push changes to the branch  in the Pr 'remove-stored-auction-response'